{"name": "Verify API handles invalid endpoint", "status": "passed", "steps": [{"name": "Given the API client is initialized for TC_01 Third Scenario", "status": "passed", "start": 1739652977295, "stop": 1739652977295}, {"name": "When I send a GET request to an invalid endpoint", "status": "passed", "start": 1739652977295, "stop": 1739652977950}, {"name": "Then the response status should be 404", "status": "passed", "start": 1739652977950, "stop": 1739652977950}], "start": 1739652977294, "stop": 1739652977950, "uuid": "7471129b-b731-4bbf-8051-77316e591b4f", "historyId": "ee47dd7b8c3888cdf3c79429f5c48e96", "fullName": "GET User List, User Login and Invalid Endpoint Test: Verify API handles invalid endpoint", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "GET User List, User Login and Invalid Endpoint Test"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}